tests/upgrade_hub_test.go

```diff
@@ -262,8 +283,8 @@ func TestHubUpgrade(t *testing.T) {
 	require.NoError(t, err, "error stopping node(s)")
 
 	// upgrade version on all nodes
-	dymension.UpgradeVersion(ctx, client, DymensionMainRepo, dymensionVersion)
-
+	// dymension.UpgradeVersion(ctx, client, DymensionMainRepo, dymensionVersion)
+	dymension.UpgradeVersion(ctx, client, DymensionMainRepo, "v4.0.0")
```

Comment:
LGTM!! Pls un-comment previous code, then we will merge it.

Reply:
Done @anhductn2001